src/olympia/amo/tests/__init__.py

```diff
@@ -644,6 +644,7 @@ def _get_created(created):
 
 def addon_factory(status=amo.STATUS_APPROVED, version_kw=None, file_kw=None, **kw):
     version_kw = version_kw or {}
+    file_kw = file_kw or {}
```

Comment:
How is this different from specifying `file_kw={}` in the function definition?

Reply:
it's a python gotcha - you can't use mutable defaults in the kwargs as they're shared with future calls to the function.